<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Gmail :: Url launch failed () :: Exception:java.net.MalformedURLException]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="3136" started-at="2016-02-24T10:46:46Z" finished-at="2016-02-24T10:46:49Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3136" started-at="2016-02-24T10:46:46Z" finished-at="2016-02-24T10:46:49Z">
      <class name="com.newt.selenium.modules.GmailLoginTest">
        <test-method status="FAIL" signature="DO()[pri:0, instance:com.newt.selenium.modules.GmailLoginTest@140e5a13]" name="DO" duration-ms="3096" started-at="2016-02-24T16:16:46Z" finished-at="2016-02-24T16:16:49Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Error occured while launching the url -   java.net.MalformedURLException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Error occured while launching the url -   java.net.MalformedURLException
	at com.newt.selenium.utlis.HelperUtility.LaunchUrl(HelperUtility.java:155)
	at com.newt.selenium.modules.GmailLoginTest.DO(GmailLoginTest.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
            <line>
              <![CDATA[Gmail :: Url launch failed () :: Exception:java.net.MalformedURLException]]>
            </line>
          </reporter-output>
        </test-method> <!-- DO -->
      </class> <!-- com.newt.selenium.modules.GmailLoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
